hazelcast-sql/src/test/java/com/hazelcast/jet/sql/impl/connector/jdbc/postgres/PostgresUpsertQueryBuilderTest.java

```diff
@@ -17,54 +17,38 @@
 package com.hazelcast.jet.sql.impl.connector.jdbc.postgres;
 
 import com.hazelcast.jet.sql.impl.connector.jdbc.JdbcTable;
-import com.hazelcast.mock.MockUtil;
-import com.hazelcast.test.HazelcastParallelClassRunner;
-import com.hazelcast.test.annotation.ParallelJVMTest;
-import com.hazelcast.test.annotation.QuickTest;
 import org.apache.calcite.sql.SqlDialect;
 import org.apache.calcite.sql.dialect.PostgresqlSqlDialect;
-import org.junit.After;
-import org.junit.Before;
-import org.junit.Test;
-import org.junit.experimental.categories.Category;
-import org.junit.runner.RunWith;
+import org.junit.jupiter.api.BeforeEach;
+import org.junit.jupiter.api.Test;
+import org.junit.jupiter.api.extension.ExtendWith;
 import org.mockito.Mock;
+import org.mockito.junit.jupiter.MockitoExtension;
 
 import java.util.Arrays;
 import java.util.Collections;
 
 import static org.assertj.core.api.Assertions.assertThat;
 import static org.mockito.Mockito.when;
-import static org.mockito.MockitoAnnotations.openMocks;
 
-@RunWith(HazelcastParallelClassRunner.class)
-@Category({QuickTest.class, ParallelJVMTest.class})
-public class PostgresUpsertQueryBuilderTest {
+@ExtendWith(MockitoExtension.class)
```

Comment:
What happened to:
```
@RunWith(HazelcastParallelClassRunner.class)
@Category({QuickTest.class, ParallelJVMTest.class})
```
?

Reply:
This is a small test class. There is no need to run in parallel. Same like other query builder tests in can run sequentially

Reply:
> This is a small test class. There is no need to run in parallel. Same like other query builder tests in can run sequentially

Ok - what about QuickTest?

I would prefer parallel tests unless this one is slower using it - as that would help improve the overall test speed…?

Reply:
@JackPGreen I think only two annotations are taken into account as ctategory. These are SlowTest and NightlyTest. The QuickTest is not taken into account.

Reply:
> @JackPGreen I think only two annotations are taken into account as ctategory. These are SlowTest and NightlyTest. The QuickTest is not taken into account.

I understand `QuickTest` is a bit pointless, but at the moment my understanding is that all the tests have it.
Removing `QuickTest` was investigated in https://github.com/hazelcast/hazelcast/pull/19266 and abandoned.
I'd be happy to try removing `QuickTest`, but across the system rather than piecemeal.

Reply:
@Category is a JUnit4 annotation. We should not mix JUnit5 and JUnit4 tags within the same test. The corresponding annotatin in JUnit5 is @Tag. https://www.baeldung.com/junit-5-migration But this annotation requires a string like @Tag("QuickTest"). If it really makes you unhappy I can revert PostgresUpsertQueryBuilderTest. The other tests did not have any annotations since the beginning. And there are many tests without any annotation. So the @Quick rule is not consistent. Do you prefer to revert the PostgresUpsertQueryBuilderTest ?

Reply:
> @category is a JUnit4 annotation. We should not mix JUnit5 and JUnit4 tags within the same test. The corresponding annotatin in JUnit5 is @tag. https://www.baeldung.com/junit-5-migration But this annotation requires a string like @tag("QuickTest"). If it really makes you unhappy I can revert PostgresUpsertQueryBuilderTest. The other tests did not have any annotations since the beginning. And there are many tests without any annotation. So the @Quick rule is not consistent. Do you prefer to revert the PostgresUpsertQueryBuilderTest ?

Ok - it looks like the QuickTest annotation went missing in the JUnit5 migration. I'm happy in that case, because JUnit5 is aresome.